pkg/security/podsecuritypolicy/selinux/mustrunas.go

```diff
@@ -21,14 +21,14 @@ import (
 	"sort"
 	"strings"
 
+	policy "k8s.io/api/policy/v1beta1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	api "k8s.io/kubernetes/pkg/apis/core"
-	"k8s.io/kubernetes/pkg/apis/policy"
 	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/util"
 )
 
 type mustRunAs struct {
-	opts *policy.SELinuxStrategyOptions
```

Comment:
it's involving unnecessary dependency. if we keep this, this would be the last place referencing internal psp package.  for a complete switching, i think we should slightly refactor here

Reply:
switching from internal to external is fine, the change from SELinuxStrategyOptions to SELinuxOptions is what surprised me. I see now the mismatch in what we get from the PSP object and what the Generate() function returns (and the comparisons made in Validate()). I guess the conversion here in construction is ok